blender_plugin/websocket_client.py

```
@@ -50,104 +50,90 @@ def _run_loop(self):
+                        raw = await asyncio.wait_for(ws.recv(), timeout=1.0)
+                        msg = json.loads(raw)
+                        self.incoming.put(msg)
+                    except asyncio.TimeoutError:
+                        continue
+                    except websockets.ConnectionClosed as e:
+                        self.last_close_code = e.code
+                        self.last_close_reason = e.reason or ""
+                        state.evicted = (e.code == EVICTED_CLOSE_CODE)
+                        state.connected = False
+                        break
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)